spec/rspec/support/method_signature_verifier_spec.rb

```diff
@@ -129,7 +129,7 @@ def arity_kw(x, y:1, z:2); end
               expect(valid?(nil, :a => 1)).to eq(false)
             end
 
-            it 'mentions the invalid keyword args in the error', :pending => RUBY_ENGINE == 'jruby' do
+            it 'mentions the invalid keyword args in the error' do
```

Comment:
I suspect this change is only appropriate for jruby-head and not for stable released JRuby, right?  Were you planning on us merging with this?